### ISSUE TITLE
[Question] How do I set up a "DSN"?

### ISSUE THREAD

**Issue by yaegassy** [modeldba/sql-autocomplete#2]
> Generate valid autocomplete suggestions for keywords, tables, or columns.

The description says table names and columns are also supported, but how do I set up the DSN?

**Comment by mtriff** [comment]
Previously, we only told you if a column or table name was a valid candidate for the location.

I've just added the ability to specify table and column names either during construction, like so:
```
  const autocompleter = new SQLAutocomplete(SQLDialect.MYSQL, ['table1'], ['columnA']);
```

Or, using the following methods:
```
  autocompleter.setTableNames(['table1', 'table2']);
  autocompleter.setColumnNames(['columnA', 'columnB']);
```

This is available in version 1.1.0. Please let me know if this is what you were looking for.

**Comment by yaegassy** [comment]
@mtriff I was thinking of using such a `DSN` to actually "connect" to the database and use the column information of the table.

```
mysql://user:password@127.0.0.1:3306/database
```

I understood that DSNs are not supported.

Thanks for the reply, This issue is closed.
